### DISCUSSION TITLE
render right to left

### DISCUSSION THREAD

**Discussion by chuckgmthompson** [Drarig29/brackets-viewer.js#106] [Q&A] [answered]
Is there a way for the bracket to be rendered in a right to left style?  Similar to how ESPN does the NCAA march madness tournament, there are 4 individual pieces of the master bracket.  2 going from left to right and 2 going from right to left.  Is it possible to render the bracket from right to left?

**Comment by Drarig29** [comment, marked as answer]
No it's not possible to do so

**Reply by ghost** [reply]
Hello, this is my interest also, what are the first steps if I want to make my computer Right to left style? Thank you!

**Reply by Drarig29** [reply]
Can you at least show a screenshot?

**Reply by chuckgmthompson** [reply]
![Screenshot 2024-04-10 092508](https://github.com/Drarig29/brackets-viewer.js/assets/26572881/91a9dbe7-9c61-483a-903a-491b3b35fdd6)

here is a screen shot of what I was asking for.  its really 4 individual brackets, which is easy but having the ones on the right, go from right to left (#2 and #4) is not supported

**Reply by Drarig29** [reply]
You can maybe do it with CSS only. Browsers are supposed to support right-to-left layouts because of some languages.

Not sure though, tell me if you manage to do it. It would be a good theme
